src/RetryMiddleware.php

```diff
@@ -54,6 +54,11 @@ public static function createDefaultDecider($maxRetries = 3)
             ResultInterface $result = null,
             $error = null
         ) use ($maxRetries) {
+            // Allow command-level options to override this value
+            $maxRetries = null !== $command['retries'] ?
```

Comment:
I think this should be `@retries` as it is a meta parameter that is not part of the API. We are already using `@` to designate other parameters as SDK related rather than API related (e.g., `@http`), so I think it makes sense.